src/components/BookForm.js

```diff
@@ -32,8 +39,32 @@ function BookForm() {
             onChange={(e) => setAutor({ author: e.target.value })}
           />
         </label>
+        {' '}
+        <label htmlFor="catagories">
+          <select
+            name="catagories"
+            required
+            onChange={(e) => setCategory({ category: e.target.value })}
+          >
+            {
+              categories.map((category) => (
+                <option
+                  key={category.id}
+                  value={category.category}
+                >
+                  {category.category}
+
+                </option>
+              ))
+        }
+          </select>
+        </label>
+        {' '}
         <input type="submit" value="ADD BOOK" />
```

Comment:
- [ ] Based on the requirements of this project, this should be a button **<Button>**. The button should include an event handler to add a book to the `books` array(with attributes _title_ and _author_)

Reply:
thanks for the feedback changed to button